### PR TITLE
feat: USD pegged MANA asset type + Audit Corrections

### PR DESCRIPTION
**Pull request by fzavalia**

### CODE REVIEW

File: .vscode/settings.json
Comment (nachomazzara) [inline .vscode/settings.json:1]: Is this file required?

Comment (fzavalia) [reply .vscode/settings.json:1]: I can remove it from this PR, but it is needed for vscode to respect the foundry config when formatting

File: src/marketplace/DecentralandMarketplaceEthereum.sol [hunk 1]
```diff
@@ -84,4 +144,38 @@ contract DecentralandMarketplaceEthereum is DecentralandMarketplaceEthereumAsset
...
+        int256 manaEthRate = _getRateFromAggregator(manaEthAggregator, manaEthAggregatorTolerance);
+
+        // Obtains the price of ETH in USD.
+        int256 ethUsdRate = _getRateFromAggregator(ethUsdAggregator, ethUsdAggregatorTolerance);
+
+        // With the obtained rates, we can calculate the price of MANA in USD.
+        int256 manaUsdRate = (manaEthRate * ethUsdRate) / 1e18;
+
+        // Updates the asset with the new values.
+        _asset = _updateAssetWithConvertedMANAPrice(_asset, manaAddress, manaUsdRate);
```
Comment (nachomazzara) [inline src/marketplace/DecentralandMarketplaceEthereum.sol:160]: maybe `value` is a better name rather than `_asset`

Comment (fzavalia) [reply src/marketplace/DecentralandMarketplaceEthereum.sol:160]: I'm updating the Asset object, which internally contains the value
